src/pages/ReviewResults.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {div, h} from 'react-hyperscript-helpers';
+import {AccessReviewHeader} from './access_review/AccessReviewHeader';
+import {DAR, DataSet, Election, Researcher} from '../libs/ajax';
+import {Notifications} from '../libs/utils';
+import * as fp from 'lodash/fp';
+import {DarApplication} from "./access_review/DarApplication";
+
+const SECTION = {
+  margin: '20px',
+};
+
+class ReviewResults extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.props = props;
+  }
+
+  componentDidMount()  {
+    const darId = this.props.match.params.referenceId;
+    this.getData(darId);
+  }
+
+  async getData(darId) {
+
+    //get information on datasets, consent, researcher, and access request
+    const getDarData = async (darId) => {
+      let datasets;
+      let darInfo;
+      let consent;
+      let researcherProfile;
+
+      try {
+        darInfo = await DAR.getPartialDarRequest(darId);
+        const researcherPromise = await Researcher.getResearcherProfile(darInfo.userId);
+        const datasetsPromise = darInfo.datasetIds.map((id) => {
+          return DataSet.getDataSetsByDatasetId(id);
+        });
+        const consentPromise = await DAR.getDarConsent(darId);
+        [consent, datasets, researcherProfile] = await Promise.all([
+          consentPromise,
+          Promise.all(datasetsPromise),
+          researcherPromise
+        ]);
+
+      } catch (error) {
+        Notifications.showError({text: 'Error initializing DAR Data'});
+        return Promise.reject(error);
+      }
+
+      return {datasets, darInfo, consent, researcherProfile};
+    };
+
+
+    //get information on access election and research purpose election
+    const getElectionInformation = async (darId) => {
+      let accessElection;
+      let accessElectionReview;
+      let rpElectionReview;
+
+      try {
+        accessElection = await Election.findElectionByDarId(darId);
+        console.log(accessElection);
+      } catch (error) {
+        //access election is null
```

Comment:
this is addressing the case for DARs with unreviewed, canceled, or closed elections. catch the error and send an information notification letting the user know why there is no vote data available

Reply:
I think we need a new story on consent to address this. See [describeDataRequestElection](https://github.com/DataBiosphere/consent/blob/7192e7f94308b986dea454d5dda47da5687f3703/src/main/java/org/broadinstitute/consent/http/service/DatabaseElectionAPI.java#L218) where we explicitly request an open election. This method should not be so restrictive and instead return the most recent DAR election if there are any. The status doesn't have to be 'OPEN' for this to work correctly.

Reply:
Okay, I will make a ticket!